### PR TITLE
Fix failures on master due to package update

### PR DESCRIPTION
**Pull request by thivya-deriv**
Card: https://app.clickup.com/t/20696747/BOT-1733
Description: Quick fix to update locaters for QuickStrategies.cy.js  and  RunMartingaleOld.cy.js  test run.

### CODE REVIEW

File: cypress/e2e/wip/dbot/pageobjects/run_panel.js [hunk 1]
```diff
@@ -38,6 +31,21 @@ class RunPanel {
...
+      .last()
+      .parents('.data-list__row--wrapper')
+      .parent()
+      .prev()
+      .within(($el) => {
+        cy.findByTestId('dt_transactions_item').should(
+          'contain.text',
+          '2.00 USD'
+        )
+      })
```
Comment (bilalmajid-deriv) [inline cypress/e2e/wip/dbot/pageobjects/run_panel.js:47]: a very length locator can we not have something simpler if not then will be Approving for now but will need to get a better locator for this element.

**Review by abuzar-deriv — APPROVED** [review]

File: cypress/e2e/wip/dbot/pageobjects/run_panel.js (hunk 1 above)
Comment (bilalmajid-deriv) [reply cypress/e2e/wip/dbot/pageobjects/run_panel.js:47]: apart from that can you also make sure that the value 2.00 USD is sent as a parameter to the function thanks

**Review by bilalmajid-deriv — APPROVED** [review]

File: cypress/e2e/wip/dbot/pageobjects/run_panel.js (hunk 1 above)
Comment (thivya-deriv) [reply cypress/e2e/wip/dbot/pageobjects/run_panel.js:47]: Here we are trying to the the amount after the first loss and also ensuring its "2", hence why it is slightly long, will try to improve this after the bot dev team updates locaters

**Review by meeyun-deriv — APPROVED** [review]
LGTM

**Pull request merged**